contracts/multisig-prover/src/encoding/mod.rs

```diff
@@ -7,4 +8,5 @@ use cosmwasm_schema::cw_serde;
 pub enum Encoder {
     Abi,
     Bcs,
+    Solana
```

Comment:
I agree with this name as a beginning, but i think at some point we sould use something more generic ?

Reply:
yea I think we should just call it `Rkyv` encoding because it is not Solana specific

Reply:
FIxed at 3812bbe263f5f7eb698e47756343d757602c72cf